### DISCUSSION TITLE
Deploy to Digital Ocean

### DISCUSSION THREAD

**Discussion by dajma00** [microsoft/aspire#6731] [Q&A]
I was able to add Aspire to my Blazor server project for testing purposes. Now, can I deploy to Digital Ocean? Any guidance on how to deploy to our own VPS server? At this time, when we push our code, we run a Github action that compiles and deploys our application to our server. After Aspire, I am guessing that we need to deploy the AppHost and Service projects as well?

**Comment by davidfowl** [comment]
There's no tooling that we ship for deploying to anything but azure. However https://github.com/prom3theu5/aspirational-manifests supports, docker compose, and kubernetes.

**Reply by davidfowl** [reply]
Are you using docker compose?
